karma.conf.js

```diff
@@ -47,9 +54,12 @@ var staticConfig = {
   }
 };
 
-var runner = function (config) {
+function runner (config) {
+  var frameworks = staticConfig.frameworks;
+  if (~staticConfig.browsers.indexOf('PhantomJS')) frameworks.push('phantomjs-shim');
   config.set(Object.assign({}, staticConfig, {
-    logLevel: config.LOG_INFO
+    logLevel: config.LOG_INFO,
+    frameworks: frameworks
```

Comment:
Same as https://github.com/recurly/recurly-js/pull/497/files#r239563021 but for outside of CI